File: applets/src/org/rr0/ufoathome/view/draw/AnimationEvent.java
package org.rr0.ufoathome.view.draw;

import java.util.Calendar;
import java.util.EventObject;

/**
 * @author Jerôme Beau
 * @version 30 janv. 2004 20:41:33
 *          $Revision: 1.1 $
 */
public class AnimationEvent extends EventObject {
    private Calendar time;

    public Calendar getTime() {
        return time;
    }

    public AnimationEvent(Object source, Calendar currentTime) {
        super(source);
        this.time = currentTime;
    }
}

File: applets/src/org/rr0/ufoathome/view/draw/AnimationEvent.ts
import {EventObject} from "../gui/EventObject";
import {Calendar} from "../../Calendar";

export class AnimationEvent extends EventObject {
  private time: Calendar;

  public getTime(): Calendar {
    return this.time;
  }

  constructor(source: Object, currentTime: Calendar) {
    super(source);
    this.time = currentTime;
  }
}
